tests/fmt_print/tests/01-run.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
```

Comment:
remove newline

Reply:
removing this newline is a pep8 violation:
```
pep8 tests/fmt_print/tests/01-run.py 
tests/fmt_print/tests/01-run.py:7:1: E402 module level import not at top of file
tests/fmt_print/tests/01-run.py:9:1: E302 expected 2 blank lines, found 1
```